client/src/redux/settings/settings-sagas.js

```diff
@@ -113,6 +113,8 @@ function* updateMySoundSaga({ payload: update }) {
 function* updateMyThemeSaga({ payload: update }) {
   try {
     const { data } = yield call(putUpdateMyTheme, update);
+    const theme = { ...data }.variables.theme;
+    localStorage.setItem('theme', theme);
```

Comment:
```suggestion
    localStorage.setItem('theme', update.theme);
```
The message coming back from the api does not include the theme.